src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Colors getRandomColor() {
+        Random random = new Random();
+        int colorIndex = random.nextInt(Colors.values().length);
+
+        return Colors.values()[colorIndex];
     }
```

Comment:
Consider making the Random instance a private final field of the ColorSupplier class, instead of creating a new instance every time the getRandomColor() method is called. This will improve performance as you will reuse the same Random instance.